devel/jmeter/files/JMeter.app/Contents/MacOS/main.command

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+@@PREFIX@@/share/jmeter/bin/jmeter &
```

Comment:
It should be `exec` so that the `sh` process does not stick around, and I assume it should not be `&` because you don't want the user to be able to open the "app" multiple times and get multiple copies of jmeter running that way.

```suggestion
exec @@PREFIX@@/share/jmeter/bin/jmeter
```